src/main/groovy/com/avast/gradle/dockercompose/ServiceInfo.groovy

```diff
@@ -5,16 +5,14 @@ import groovy.transform.Immutable
 @Immutable
 class ServiceInfo {
     String name
-    ServiceHost serviceHost
-    /* Mapping from exposed to forwarded port. */
-    Map<Integer, Integer> tcpPorts
-    String containerHostname
-    /* Docker inspection */
-    Map<String, Object> inspection
-    String getContainerId() { inspection.Id }
+    List<ServiceInstanceInfo> serviceInstanceInfos = []

```

Comment:
What about changing it from `List<ServiceInstanceInfo>` to `Map<String, ServiceInstanceInfo>`? It would simplify the usage on few places (and also in the public API as described in README).